### PR TITLE
feat: rename crypto crate

### PR DESCRIPTION
**Pull request by alexfertel**
Decided to go for `openzeppelin-crypto-rs`, but obv open to something different.

UPDATE: In the end went for `openzeppelin-crypto`.

### CODE REVIEW

File: lib/crypto/Cargo.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [package]
-name = "crypto"
+name = "openzeppelin-crypto-rs"
```
Comment (qalisander) [inline lib/crypto/Cargo.toml:2]: Isn't it contradicting to the [official guidelines](https://rust-lang.github.io/api-guidelines/naming.html)?

> Crate names should not use -rs or -rust as a suffix or prefix. Every crate is Rust! It serves no purpose to remind users of this constantly.

Comment (qalisander) [reply lib/crypto/Cargo.toml:2]: Of course there are many crates that doesn't follow that convention. But seems it is an official recomendation

Comment (alexfertel) [reply lib/crypto/Cargo.toml:2]: Yes! I'll remove the suffix, was thinking something else. thx

Comment (bidzyyys) [reply lib/crypto/Cargo.toml:2]: I am not a fun of this suffix; I vote for just  `openzeppelin-crypto`.

**Review by bidzyyys — COMMENTED** [review]
I would leave `openzeppelin-crypto`

File: lib/crypto/Cargo.toml (hunk 1 above)
Comment (qalisander) [reply lib/crypto/Cargo.toml:2]: `openzeppelin-crypto` yeah. This one seems good also for me

**Review by bidzyyys — APPROVED** [review]
LGTM!

**Review by ggonzalez94 — APPROVED** [review]

**Pull request merged**